webpack/components/ProxmoxServer/components/NetworkInterface.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Title, Divider, PageSection } from '@patternfly/react-core';
+import InputField from '../../common/FormInputs';
+import ProxmoxComputeSelectors from '../../ProxmoxComputeSelectors';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const NetworkInterface = ({
+  id,
+  networks,
+  bridges,
+  data,
+  updateNetworkData,
+}) => {
+  const [network, setNetwork] = useState(data);
+
+  useEffect(() => {
+    const currentNetData = JSON.stringify(network);
+    const parentNetData = JSON.stringify(data);
+
+    if (currentNetData !== parentNetData) {
+      updateNetworkData(id, network);
+    }
+  }, [network, id, data, updateNetworkData]);
+  const handleChange = e => {
+    const { name, type, checked } = e.target;
+    const value = type === 'checkbox' ? (checked ? '1' : '0') : e.target.value;
```

Comment:
I remember some linter complaining about nested ternary operators :D

Maybe you could simplify the code and make it easier to read.

Reply:
fixed it